raster_feeder/nrr/move.py

```diff
@@ -58,11 +58,7 @@ def move(time_name, source_name, target_name):
     target = get_store(time_name=time_name, store_name=target_name)
 
     # promote
-    locker = turn.Locker(
```

Comment:
Maybe good to leave the possibility of reconfiguring the `TURN` redis through the config?

Reply:
Are you referring to the removed `db` and `password` options or something else? If the former, I figured this will not be used much in the future and all we use currently is this...

Reply:
Ok?

Reply:
Ok!